addons/web/static/tests/views/pivot_view.test.js

```diff
@@ -3767,3 +3767,19 @@ test("middle clicking on a cell triggers a doAction", async () => {
     expect("table").toHaveClass("o_enable_linking");
     await contains(".o_pivot_cell_value:eq(1)").click({ ctrlKey: true }); // should trigger a do_action
 });
+
+test("display '0' for empty int field values in grouped pivot view", async () => {
+    Partner._records[0].foo = false;
```

Comment:
```suggestion
    Partner._records[0].foo = 0;
```

Reply:
Hello @aab-odoo ,

If we set it to 0, the test case pass without our condition, because the condition is being checked based on the value rather than the display name. 

Thank you :slightly_smiling_face:

Reply:
Ho indeed. I'm checking with framework python because it's different from web_read then.